parts/k8s/kubernetesinstalls.sh

```diff
@@ -16,6 +16,11 @@ removeMoby() {
     sudo apt-get purge -y moby-engine moby-cli
 }
 
+upgradeOs() {
+    apt_get_update || exit $ERR_APT_UPDATE_TIMEOUT
+    apt_get_dist_upgrade
```

Comment:
We would want to add `|| exit $ERR_APT_UPDATE_TIMEOUT` to the dist-upgrade as well.

Reply:
Do we really want to fail the entire script if this fails ?

Reply:
Yes, because we want everything in CSE to be 100% deterministic + successful, and we want to know if certain steps don't meet that criteria. If this is a "run in the background w/ best effort" kind of this we should move it out of CSE and into some other area of enforcement.

Reply:
All right.